docroot/themes/humsci/humsci_basic/src/scss/tools/_mixins.general.scss

```diff
@@ -133,7 +133,7 @@
 @mixin hb-ordered-list {
   counter-reset: li;
   list-style-type: none;
-  margin: hb-calculate-rems(28px) 0;
+  margin: 0;
```

Comment:
I'm not sure if this is related, but I noticed some spacing issues on nested ordered / unordered lists that we should look at:

<details>
<summary>Expand to see image</summary>
<img src="https://user-images.githubusercontent.com/1256329/78085768-de914e80-7389-11ea-832a-ce059fd68fb6.png" />
</details>

<details>
<summary>Expand to see markup you can paste to reproduce the issue</summary>
<pre>
<ul><li>Item 1</li>
	<li>Item 2</li>
	<li>Item 3
	<ul><li>Item A</li>
		<li>Item B</li>
		<li>Item C</li>
	</ul></li>
	<li>Item 4
	<ul><li>Item A</li>
		<li>Item B
		<ul><li>Item i</li>
			<li>Item ii</li>
		</ul></li>
		<li>Item C</li>
	</ul></li>
	<li>Item 5</li>
</ul>
<ol><li>Item 1</li>
	<li>Item 2</li>
	<li>Item 3
	<ol><li>Item A</li>
		<li>Item B</li>
		<li>Item C</li>
	</ol></li>
	<li>Item 4
	<ol><li>Item A</li>
		<li>Item B
		<ol><li>Item i</li>
			<li>Item ii</li>
		</ol></li>
		<li>Item C</li>
	</ol></li>
	<li>Item 5</li>
</ol>
</details>

Reply:
Oh I see. I'll add it back in.

Reply:
I'm going to just add a margin-bottom here because we have to plan for headings being on the top. 
<img width="699" alt="Screen Shot 2020-04-01 at 11 47 51 AM" src="https://user-images.githubusercontent.com/19614647/78157932-9f591100-740e-11ea-96a6-e1d57f8782c9.png">

Reply:
We're still going to have an issue here:

<img width="806" alt="Screen Shot 2020-04-01 at 12 25 51 PM" src="https://user-images.githubusercontent.com/1256329/78161926-2a88d580-7414-11ea-8f10-15b0930f046b.png">

In order to avoid this, I think the margins need to be different for the top-level `<ul>` / `<ol>` then they are for nested ones. Happy to pair on this, if you think it would help.

Reply:
I think this is out of scope for the work I was doing here. I can add it to my next polish card? Or I can pair on it today and get it done. What do you think?

Reply:
I'm ok with handling it in the next polish card if you want. 👍

Reply:
Sounds good. I'll add it to the card.